resources/migrations/000_migrations.yaml

```diff
@@ -3646,3 +3646,51 @@ databaseChangeLog:
             columns:
               - column:
                   name: dashboard_id
+  - changeSet:
+      id: 56
+      author: senior
+      changes:
+        - createTable:
+            tableName: metabase_dimensions
+            remarks: ''
```

Comment:
I would definitely add remarks to the new tables and columns explaining their purpose,
so potential contributors or other people trying to hack Metabase don’t ask you a million questions about why the stuff doesn’t do what they expected it to do.

Otherwise you'll have to explain that the purpose of the table is something completely different than what they assumed. Just speaking from experience, a little bit of extra dox will save you GH conversations down the road